src/apps/bristleback_apps/bristleback_apps_common/task_priorities.h

```diff
@@ -28,8 +28,8 @@
 
 #define SERIAL_TX_TASK_PRIORITY 2
 #define CONSOLE_RX_TASK_PRIORITY 2
+#define USER_TASK_PRIORITY 2
```

Comment:
Since this is used for the LPUART(user's) RX task priority, I thought it would make more sense that both serial tasks had the same priority.